src/phpDebug.ts

```diff
@@ -197,14 +213,20 @@ class PhpDebugSession extends vscode.DebugSession {
 
     /** converts a server-side XDebug file URI to a local path for VS Code with respect to source root settings */
     protected convertDebuggerPathToClient(fileUri: string): string {
-        // convert the file URI to a path
-        const serverPath = decodeURI(url.parse(fileUri).pathname.substr(1));
+        // convert the file URI to a path. Don't remove starting slash on unix platforms.
+        let n:number = (/^win/.test(process.platform)) ? 1 : 0;
+        const serverPath = url.parse(fileUri).pathname.substr(n);
```

Comment:
This is great in general, but a check for `process.platform === 'win32'` is enough (it has nothing to do with 32 / 64 bit)